src/main/java/io/lettuce/core/Transports.java

```diff
@@ -22,16 +22,19 @@
 import io.netty.channel.Channel;
 import io.netty.channel.EventLoopGroup;
 import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.DatagramChannel;
+import io.netty.channel.socket.nio.NioDatagramChannel;
 import io.netty.channel.socket.nio.NioSocketChannel;
 
 /**
  * Transport infrastructure utility class. This class provides {@link EventLoopGroup} and {@link Channel} classes for socket and
  * native socket transports.
  *
  * @author Mark Paluch
+ * @author Yohei Ueki
  * @since 4.4
  */
-class Transports {
+public class Transports {
```

Comment:
[Supplimental Comments] 
I made this class public since now this is referenced by outside of the package.
https://github.com/lettuce-io/lettuce-core/pull/1517/files#diff-724688b7234bd44bd8e5f08ce8b0a616ecaf11729ebd99ad72d510431878a6f4R60-R62